build/unicode_test.go

```diff
@@ -0,0 +1,98 @@
+package build
+
+import (
+	"testing"
+)
+
+func TestAppendRune(t *testing.T) {
+	table := &CodeTable{CodeRange{48, 57}}
+
+	table.AppendRune(45)
+	CompareTable(t, *table, "09--", false)
+
+	table.AppendRune(44)
+	CompareTable(t, *table, "09--,,", false)
+
+	table.AppendRune(95)
+	CompareTable(t, *table, "09--,,__", false)
+}
```

Comment:
These tests are great. I think they'd be even easier to understand if the used the notation for the rune values of characters `'0'`, rather than the values directly `48` (which makes me go look at an ASCII table).